aspnet/fundamentals/servers.rst

```diff
@@ -167,12 +167,12 @@ Kestrel is a cross-platform web server based on `libuv <https://github.com/libuv
 
 Learn more about working with Kestrel to create :doc:`/tutorials/your-first-mac-aspnet`.
 
+.. note:: Kestrel is not hardened for internet facing traffic and should always be used behind a secure proxy server (ex. IIS, Nginx).
```

Comment:
Rather harsh, How about "Kestrel is designed to be run behind a proxy (For example IIS or nginx) and should not be deployed directly facing the internet."